csrc/executor.cpp

```diff
@@ -257,8 +257,9 @@ void FusionExecutor::compileFusion(
   TORCH_INTERNAL_ASSERT(
       options_.device.is_cuda(), "Provided device to CUDA fuser is the CPU.");
   auto properties = at::cuda::getDeviceProperties(options_.device.index());
-  configured_device_smem_ = properties->sharedMemPerBlock;
-  device_smem_limit_ = properties->sharedMemPerBlockOptin;
+  // TODO: These properties should be set as part of the constructor so that it
+  // can be const
+  device_smem_limit_ = static_cast<int64_t>(properties->sharedMemPerBlockOptin);
```

Comment:
Naive question: is `sharedMemPerBlockOptin == properties->sharedMemPerBlock` when there's not runtime API overwrite?

Reply:
I think so.